### PR TITLE
Add 4.10->4.11 upgrade test for vsphere

### PR DESCRIPTION
**Pull request by shellyyang1989**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 1]
```diff
@@ -278,6 +278,50 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-realtime-kernel
+- as: vsphere-ipi-disconnected
+  cron: 0 8 */7 * *
+  steps:
+    cluster_profile: vsphere-discon
+    env:
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml:285]: @shellyyang1989 same comment with https://github.com/openshift/release/pull/30499/files#r921869843

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 2]
```diff
@@ -286,8 +330,21 @@ tests:
...
       UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @vsphere-ipi'
     test:
+    - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-upi-disconnected-secureboot
+  cron: 0 12 */7 * *
+  steps:
+    cluster_profile: vsphere-discon
+    env:
```
[the remark this answers is not in the document]
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml:341]: Updated. Thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml (hunk 1 above)
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml:285]: Updated. Thanks!

**Comment by kuiwang02** [comment]
lgtm for golang part

**Comment by shellyyang1989** [comment]
cc @liangxia for review and approval. Thanks!

**Pull request merged**